.gitignore

```diff
@@ -471,3 +471,5 @@ $RECYCLE.BIN/
 
 # End of https://www.toptal.com/developers/gitignore/api/python,jupyternotebooks,vim,visualstudiocode,pycharm,emacs,linux,macos,windows
 test.ipynb
+
+testfile.txt
```

Comment:
IMO test should not produce any permanent file.
So they should either be locate in a temporary directory (i.e., using the `tmp_path` fixture) or removed after the tests are exectuted (pytest has teardown capabilities).

Also, I would place additions to the default gitignore in the header of this file, where there are others additions to the default of `toptal`. Otherwise, these additions are very hard to find for other developers.

Reply:
done